dipy/workflows/tests/test_io.py

```diff
@@ -63,7 +63,8 @@ def test_io_fetch_fetcher_datanames():
                      'stanford_labels', 'stanford_pve_maps', 'stanford_t1',
                      'syn_data', 'taiwan_ntu_dsi', 'target_tractogram_hcp',
                      'tissue_data', 'qte_lte_pte', 'resdnn_weights',
-                     'DiB_217_lte_pte_ste', 'DiB_70_lte_pte_ste', "hcp"]
+                     'DiB_217_lte_pte_ste', 'DiB_70_lte_pte_ste', "hcp",
+                     "hbn"]
```

Comment:
`hbn` or `hbn_preproc`. it should be the same name as the function

Reply:
I think this is correct now.